.gitignore

```diff
@@ -244,3 +244,8 @@ video_creation/data/envvars.txt
 
 config.toml
 video_creation/data/videos.json
+results/client_secrets.json
+results/client_secrets.json.template
+results/upload_video.py-oauth2.json
+results/*/*.mp4
```

Comment:
Again, easier just to ignore `results/*`

Reply:
can't ignore everything in results as that is where the script is. so it only needs to ignore the .mp4 files